chroma-manager/tests/framework/utils/defaults.sh

```diff
@@ -102,17 +102,17 @@ set_defaults() {
     # if you use this, don't forget to update chroma-manager/chroma_agent_comms/views.py
     #LUSTRE_REVIEW_BUILD="12345"
     if [ -n "$LUSTRE_REVIEW_BUILD" ]; then
-        BASE_URL="https://build.whamcloud.com/lustre-reviews/configurations/axis-arch/\\\$basearch/axis-build_type"
+        BASE_URL="https://build.hpdd.intel.com/jobs-pub/lustre-b2_10/configurations/axis-arch/\\\$basearch/axis-build_type"
```

Comment:
Ditto on the domain name but more importantly, you replaced _lustre-reviews_ with _lustre-b2_10_ and this branch of code is specifically for being able to test a review build.